test/api.js

```
@@ -0,0 +1,97 @@
+var mongoose = require('mongoose');
+var should = require('should');
+var request = require('supertest');
+var EntryModel = require('../models/entries');
+var entries = require('../api/entries');
+var url = 'localhost:5000/api'
```

Can we throw an error and prevent running these tests on production?